packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -1129,159 +1120,6 @@ describe("Runtime", () => {
 			);
 		});
 
-		describe("[DEPRECATED] Future op type compatibility", () => {
```

Comment:
Sorry... I think many of these tests are good to keep around.  They are illustrating that things fail with an unknown type.  So I'd call this section "Unrecognized types not supported", and make sure the titles are correct for each of the tests that assert that something throws (e.g. the Resubmit one still says something about being ignored).

Of course _do_ delete the first test "can submit op compat behavior"

Reply:
And get rid of the compatDetails in each test case too, of course. That's what tripped me up about saying to delete them I think.

Reply:
The only tests that remained are Op with unrecognized type is ignored by resubmit and process remote op with unrecognized type. Those are the only ones that I found necessary but let me know if you think we should keep anything else.